save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestExecutionController.kt

```diff
@@ -16,11 +16,15 @@ import org.springframework.web.bind.annotation.RestController
 class TestExecutionController(private val testExecutionService: TestExecutionService) {
     /**
      * @param testExecutions list of test executions
-     * @return
+     * @return response
      */
     @PostMapping(value = ["/saveTestResult"])
     fun saveTestResult(@RequestBody testExecutions: List<TestExecution>): ResponseEntity<String> {
-        testExecutionService.saveTestResult(testExecutions)
+        try {
+            testExecutionService.saveTestResult(testExecutions)
+        } catch (exception: IllegalArgumentException) {
+            return ResponseEntity.ok().body("Error to save")
```

Comment:
Why `ok` if there is an error? Also, you should never silently return error and do nothing else. You definitely should log an error message in backend so that we can find out what happened here.

Reply:
But we already log error in `TestExecutionService`. Should we log here too?

Reply:
No, you are right, logging is redundant here.